Package.swift

```diff
@@ -16,6 +16,7 @@ import Foundation
 // Therefore, it's important that we only expose modules that we want customers to use. Internal modules should not be included in this array.
```

Comment:
Friendly reminder to also update cocoapods files. 

You will probably need to create a new `podspec` file, update `CustomerIO.podspec` file, and also update the CI cocoapods deployments script to deploy the new `podspec` file.

Reply:
Good call-out.